### ISSUE TITLE
What's class D_DownBlock(torch.nn.Module) mean?What's num_stages mean?

### ISSUE THREAD

**Issue by yangyingni** [alterzero/DBPN-Pytorch#45]
class D_DownBlock(torch.nn.Module):
    def __init__(self, num_filter, kernel_size=8, stride=4, padding=2, num_stages=1, bias=True, activation='prelu', norm=None)
    super(D_DownBlock, self).__init__()
    self.conv = ConvBlock(num_filter*num_stages, num_filter, 1, 1, 0, activation, norm=None)
    self.down_conv1 = ConvBlock(num_filter, num_filter, kernel_size, stride, padding, activation, norm=None)
    self.down_conv2 = DeconvBlock(num_filter, num_filter, kernel_size, stride, padding, activation, norm=None)
    self.down_conv3 = ConvBlock(num_filter, num_filter, kernel_size, stride, padding, activation, norm=None)

    Why it started with conv,then down_conv1,down_conv2,down_conv3?What's difference between  DownBlock?What's num_stages mean?Thank you.
